docs/instrumentation_examples.md

```diff
@@ -0,0 +1,214 @@
+# Armory Instrumentation Examples: Measuring Experiment Artifacts Using Probes and Meters
+For an introduction to `Probe`s and `Meter`s, please refer to [Measurement Overview](./metrics.md).
+
+## Context
```

Comment:
I think you can delete the `Context` header, the three bullets, and the sentence above the bullets. 

Originally I was thinking the third bullet may be confusing in that seems to imply you need to create some separate custom monitoring script, as opposed to dropping probes into your already-existing code (I understand there's also the `user_init` piece, but don't think we need to throw that at the reader at the jump). After removing this bullet, I think it may be more straightforward to just jump straight into `We assume the user...` followed by `Recall the steps...`

Reply:
resolved with c24837a7